docs/auth/getting-started.md

```diff
@@ -30,10 +30,10 @@ export class AppComponent {
   constructor(public auth: AngularFireAuth) {
   }
   login() {
-    this.auth.signInWithPopup(new auth.GoogleAuthProvider());
+    this.auth.auth.signInWithPopup(new auth.GoogleAuthProvider());
```

Comment:
I think that the approach in PR https://github.com/angular/angularfire/pull/2483 is better. I.e.
```ts
this.afAuth.auth.
```